### PR TITLE
5-edit-mode

### PR DESCRIPTION
**Pull request by nikita-resh**
image
- add opp to edit card
- add opp to delete card

### CODE REVIEW

File: src/states/EditDialog/EditDialog.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    if (!currentTitle?.trim() || !currentText?.trim()) {
+      setIsFilled(false);
+    } else {
+      setIsFilled(true);
+    }
+  }, [currentTitle, currentText]);
+
+  useEffect(() => {
+    setCurrentTitle(cardList.find((item) => item.id === selected)?.title);
+    setCurrentText(cardList.find((item) => item.id === selected)?.text);
```
Comment (ArtsiomLysianok) [inline src/states/EditDialog/EditDialog.js:26]: You can move out card finding logic

File: src/states/EditDialog/EditDialog.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+}) => {
+  const [currentTitle, setCurrentTitle] = useState("");
+  const [currentText, setCurrentText] = useState("");
+  const [isFilled, setIsFilled] = useState(false);
+
+  useEffect(() => {
+    if (!currentTitle?.trim() || !currentText?.trim()) {
+      setIsFilled(false);
+    } else {
+      setIsFilled(true);
```
Comment (ArtsiomLysianok) [inline src/states/EditDialog/EditDialog.js:20]: Will that works in the same way ?
setIsFilled(!!currentTitle?.trim() && !!currentText?.trim());

Comment (nikita-resh) [reply src/states/EditDialog/EditDialog.js:20]: It will

File: src/states/EditDialog/EditDialog.js (hunk 1 above)
Comment (nikita-resh) [reply src/states/EditDialog/EditDialog.js]: Done.

**Review by ArtsiomLysianok — APPROVED** [review]

**Pull request merged**